data/transactions/logic/eval.go

```diff
@@ -248,6 +248,23 @@ type boxRef struct {
 	name string
 }
 
+// EvalConstants contains data that opcodes rely on during runtime.
+type EvalConstants struct {
+	// MaxLogSize is the limit of total log size from n log calls in a program
+	MaxLogSize uint64
+
+	// MaxLogCalls is the limit of total log calls during a program execution
+	MaxLogCalls uint64
+}
+
+// NewRuntimeEvalConstants gives a set of const params used in normal runtime of opcodes
+func NewRuntimeEvalConstants() EvalConstants {
```

Comment:
nit: I believe the `New` prefix implies returning a pointer in go? I think just `RuntimeEvalConstants` would be sufficient

Reply:
see [refactor Simulate interface, remove unnecessary codec struct, remove …](https://github.com/algorand/go-algorand/pull/5247/commits/61d8b7e19f08848a812d01c8933950740c97b60d)